elastic/logs/track.json

```diff
@@ -21,6 +21,7 @@
 {% set p_query_time_period = (query_time_period | default(900)) %}
 {% set p_query_request_params = (query_request_params | default({}))%}
 {% set p_include_esql_queries = (include_esql_queries | default(build_flavor != "serverless")) %}
+{% set p_synthetic_source_keep = (synthetic_source_keep | default) %}
```

Comment:
That's unusual use of `default` function. Can you define the default value explicitly with `default(<SOME-VALUE>)`? It will be easier to read later on. I think this default resolves to an empty string which is not obvious.

Reply:
I wanted to make explicit the fact that the default value is not set...maybe I am too explicit here?

Reply:
You can use `default("")` or resign from `p_synthetic_source_keep` similarly to `index_mode` which does not have a corresponding `p_index_mode`. Let's make those 2 behave similar.